apmpackage/apm/docs/README.md

```diff
@@ -0,0 +1,684 @@
+# APM Integration
```

Comment:
Does this generated file need to be committed?

Reply:
If you don't intend to verify it, then I suppose that it doesn't. Currently, it looks like it fails `check-git-diff` when uncommited. Let me know what you think.

Reply:
Maybe you ran `elastic-package build` in the apmpackage/apm source directory while making changes? Normally it would be built from `build/apmpackage`. I just checked out your branch, removed the file, and running `make build-package` and `make build-package-snapshot`, and the file wasn't recreated. So please do remove it, and let's see if anything fails in CI.

Reply:
Removed.